runtime/JavaScript/src/antlr4/error/DiagnosticErrorListener.d.ts

```diff
@@ -1,9 +1,16 @@
+import {ATNConfigSet} from "../atn";
+import {BitSet} from "../misc/BitSet";
+import {DFA} from "../dfa";
 import {ErrorListener} from "./ErrorListener";
 import {Recognizer} from "../Recognizer";
 import {RecognitionException} from "./RecognitionException";
 
 export declare class DiagnosticErrorListener<TSymbol> implements ErrorListener<TSymbol> {
+    reportAmbiguity(recognizer: Recognizer<TSymbol>, dfa: DFA, startIndex: number, stopIndex: number, exact: boolean, ambigAlts: BitSet, configs: ATNConfigSet): void;
```

Comment:
since DiagnosticErrorListener inherits from ErrorListener, I don't think adding these prototypes is useful. What happens if you don't ?

Reply:
done